src/gowrapper/ckks/params.go

```diff
@@ -63,6 +64,37 @@ func lattigo_getDefaultPQParams(paramEnum uint8) Handle6 {
 	return marshal.CrossLangObjMap.Add(unsafe.Pointer(&params))
 }
 
+//export lattigo_newParameters
+func lattigo_newParameters(logN uint64, qi *C.constULong, numQi uint8, pi *C.constULong, numPi uint8, logScale uint8) Handle6 {
+	size := unsafe.Sizeof(uint64(0))
+
+	Qi := make([]uint64, numQi)
+	qiPtr := uintptr(unsafe.Pointer(qi))
+	for i := range Qi {
+		Qi[i] = *(*uint64)(unsafe.Pointer(qiPtr + size*uintptr(i)))
+	}
+
+	Pi := make([]uint64, numPi)
+	piPtr := uintptr(unsafe.Pointer(pi))
+	for i := range Pi {
+		Pi[i] = *(*uint64)(unsafe.Pointer(piPtr + size*uintptr(i)))
+	}
+
+	var rlweParams rlwe.Parameters
+	var err error
+	rlweParams, err = rlwe.NewParameters(int(logN), Qi, Pi, rlwe.DefaultSigma)
```

Comment:
Nit: why sometimes use short variable declaration `:=` but sometimes use `var`.  A little inconsistent.

Reply:
I tried to use `var` where possible, but in some cases I don't know what the variable type is, e.g., because I copied sample code from elsewhere.